packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/delete.module.js

```diff
@@ -17,7 +17,15 @@ angular
     'pascalprecht.translate',
   ])
   .config(routing)
-  .run(/* @ngTranslationsInject:json ./translations */)
-  .component('dedicatedCloudDatacenterBackupDelete', component);
+  .component('dedicatedCloudDatacenterBackupDelete', component)
+  .run(
+    /* @ngInject */ ($translate, $transitions) => {
+      $transitions.onBefore(
```

Comment:
Isn't this already applied on the whole app ?

Reply:
Its currently not added. I have moved this to the app level :)
Thanks for the suggestion